projects/storefrontlib/src/cms-components/misc/icon/icon-loader.service.ts

```diff
@@ -34,6 +35,14 @@ export class IconLoaderService {
     }
   }
 
+  /**
+   * Indicates the icon direction (ltr vs rtl). The icon direction is configurable, but
+   * optional, as only a few icons should be flipped for rtl direction.
```

Comment:
The description is a bit misleading to me.

If I understood well the implementation, we return the layout direciton, which makes the icon to flip. Not the _direction of the icon_ itself.

Reply:
yeah, not easy to describe. i did another attempt.